pom.xml

```diff
@@ -88,7 +88,8 @@ under the License.
     <argLine>-ea -Xmx4g -Xss4m ${extraJavaTestArgs}</argLine>
     <additional.3_3.test.source>spark-3.3-plus</additional.3_3.test.source>
     <additional.3_4.test.source>spark-3.4</additional.3_4.test.source>
-    <shims.source>spark-3.x</shims.source>
+    <shims.majorSource>spark-3.x</shims.majorSource>
+    <shims.minorSource>spark-3.4</shims.minorSource>
```

Comment:
Hmm, I don't get the naming? What `majorSource` and `minorSource` mean?

Reply:
How about `majorVerSrc` and `minorVerSrc`?

Reply:
Fine by me, done in [112c7c6](https://github.com/apache/datafusion-comet/pull/342/commits/112c7c61db86f9a7383b0667af846ac02b1703c7)